src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -76,21 +76,29 @@
            on-import (:on-import props)]
        (assert selected-config (str "Missing a selected configuration: " props))
        (assert on-import (str "Missing on-import handler: " props))
-       (assert workspace-id (str "Missing workspace-id: " props))
        [:div {}
-        (create-formatted-header "Import Method Configuration")
+        [comps/Button {:icon :angle-left
```

Comment:
Use absolute positioning to put it in the corner of the whole dialog, and not push the rest of the content down.

Another spot for it might be 50% of the way down the left side.